add exclude "src/linux/LinuxVST3Helpers.cpp" to vst2 section in premake.lua

As title.
All files in src/linux are added to vst2 build in function plugincommon() in premake.lua
so add
excludes {
	      "src/linux/LinuxVST3Helpers.cpp"
        }

somewhere in the vst2 section to allow the vst2 plugin to build without error

The error was in the link stage.
Linking surge-vst2
obj/x64/Release/surge-vst2/LinuxVST3Helpers.o: In function `_GLOBAL__sub_I_LinuxVST3Helpers.cpp':
LinuxVST3Helpers.cpp:(.text.startup._GLOBAL__sub_I_LinuxVST3Helpers.cpp+0x3b): undefined reference to `Steinberg::Singleton::isTerminated()'
LinuxVST3Helpers.cpp:(.text.startup._GLOBAL__sub_I_LinuxVST3Helpers.cpp+0x44): undefined reference to `Steinberg::Singleton::lockRegister()'
LinuxVST3Helpers.cpp:(.text.startup._GLOBAL__sub_I_LinuxVST3Helpers.cpp+0x6b): undefined reference to `Steinberg::UpdateHandler::UpdateHandler()'
LinuxVST3Helpers.cpp:(.text.startup._GLOBAL__sub_I_LinuxVST3Helpers.cpp+0x7e): undefined reference to `Steinberg::Singleton::registerInstance(Steinberg::FObject**)'
LinuxVST3Helpers.cpp:(.text.startup._GLOBAL__sub_I_LinuxVST3Helpers.cpp+0x56): undefined reference to `Steinberg::Singleton::unlockRegister()'
collect2: error: ld returned 1 exit status

--- a/src/linux/LinuxVST3Helpers.cpp
+++ b/src/linux/LinuxVST3Helpers.cpp
@@ -2,6 +2,7 @@
 ** The various things we need to help with VST3 Linux
 */
 
+#if TARGET_VST3
 #include <iostream>
 #include <pthread.h>
 #include <unistd.h>
@@ -218,3 +219,4 @@ void LinuxVST3FrameOpen(VSTGUI::CFrame* that, void* parent, const VSTGUI::Platfo
 
    IdleUpdateHandler::start();
 }
+#endif

